mcp/event_test.go

```diff
@@ -287,7 +287,7 @@ func BenchmarkMemoryEventStore(b *testing.B) {
 			payload := make([]byte, test.datasize)
 			start := time.Now()
 			b.ResetTimer()
-			for i := 0; i < b.N; i++ {
+			for i := range b.N {
```

Comment:
b.Loop? Or is that in too recent a version?

Reply:
Too recent, until we update our go directive.